nibabel/tests/test_round_trip.py

```diff
@@ -181,14 +181,14 @@ def check_arr(test_id, V_in, in_type, out_type, scaling_type):
             rel_mx_e = rel_err[abs_fails].max()
         else:
             rel_mx_e = None
-        print (test_id,
-               np.dtype(in_type).str,
-               np.dtype(out_type).str,
-               exp_abs_mx_e,
-               abs_mx_e,
-               rel_thresh,
-               rel_mx_e,
-               slope, inter)
+        print(test_id,
+              np.dtype(in_type).str,
+              np.dtype(out_type).str,
+              exp_abs_mx_e,
+              abs_mx_e,
+              rel_thresh,
+              rel_mx_e,
+              slope, inter)
```

Comment:
Looks like this was supposed to print a tuple but got lucky in 2-3 conversion.

```suggestion
        print((test_id,
               np.dtype(in_type).str,
               np.dtype(out_type).str,
               exp_abs_mx_e,
               abs_mx_e,
               rel_thresh,
               rel_mx_e,
               slope, inter))
```

Reply:
Nice, I thought it's just python 3 with a space after the print (`print ('hi')` works)